server_admin/topics/authentication/kerberos.adoc

```diff
@@ -188,6 +188,48 @@ WARNING: Credential delegation has some security implications so only use it if
          It's highly recommended to use it together with HTTPS.
          See for example http://www.microhowto.info/howto/configure_firefox_to_authenticate_using_spnego_and_kerberos.html[this article] for more details.
 
+==== Cross-realm trust
+
+In the Kerberos V5 protocol, the `realm` is a set of Kerberos principals defined in the Kerberos database (typically LDAP servers).
+Kerberos protocol has a concept of cross-realm trust. For example if there are 2 kerberos realms A and B, the cross-realm trust
+will allow the users from realm A to access resources (services) of realm B. This means that realm B trusts the realm A.
+
+.Kerberos cross-realm trust
+image:images/kerberos-trust-basic.png[]
+
+
+The {project_name} server has the support for cross-realm trust. There are few things which need to be done to achieve this:
+
+* Configure the Kerberos servers for the cross-realm trust. This step is dependent on the concrete Kerberos server implementations used.
+In general, it is needed to add the Kerberos principal `krbtgt/B@A` to both Kerberos databases of realm A and B. It is needed that
+this principal has same keys on both Kerberos realms. This is usually achieved when the principals have same password, key version number
+and there are same ciphers used in both realms. It is recommended to consult the Kerberos server documentation for more details.
+
+NOTE: The cross-realm is unidirectional by default. If you want it bidirectional and support that realm A trusts realm B,
+you need to add also the principal `krbtgt/A@B` to both Kerberos databases.
+However trust is transitive by default. If realm B trusts realm A and realm C trusts realm B, then realm C automatically trusts
+realm A as well without a need to have principal `krbtgt/C@A` available. Some additional configuration (for example `capaths`)
+may be needed to configure on Kerberos client side, so that the clients are able to find the trust path. Again, consult Kerberos
+documentation for more details.
```

Comment:
NOTE: The cross-realm trust is unidirectional by default. If you want bidirectional trust to have realm A also trust realm B,
you must also add the principal `krbtgt/A@B` to both Kerberos databases. However, trust is transitive by default. If realm B trusts realm A and realm C trusts realm B, then realm C automatically trusts realm A without a need to have principal `krbtgt/C@A` available. Some additional configuration (for example `capaths`) may be needed to configure on Kerberos client side, so that the clients are able to find the trust path. Consult the Kerberos documentation for more details.